multicloud/src/main/java/de/uniulm/omi/cloudiator/sword/multicloud/pricing/aws/AWSProduct.java

```diff
@@ -0,0 +1,12 @@
+package de.uniulm.omi.cloudiator.sword.multicloud.pricing.aws;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.Map;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class AWSProduct {
+    public String sku;
```

Comment:
all field public?

Reply:
I'm gonna leave it as is - it's just a pojo